app/(dashboard)/mailboxes/[mailbox_slug]/[category]/conversation/conversation.tsx

```
@@ -406,16 +410,25 @@ const ConversationContent = () => {
   const conversationMetadata = {
     emailFrom: (
       <div className="flex items-center gap-3">
-        <Tooltip open>
+        <Tooltip open={emailCopied}>
           <TooltipTrigger asChild>
             <div
               onClick={copyEmailToClipboard}
-              className="lg:text-base text-sm text-foreground responsive-break-words truncate cursor-pointer hover:text-primary"
+              onKeyDown={(e) => {
+                if (e.key === 'Enter' || e.key === ' ') {
```

ESLint

[error] Replace `'Enter'·||·e.key·===·'·'` with `"Enter"·||·e.key·===·"·"`

(prettier/prettier)